src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure [] figures = new Figure [6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
+        for (int i = 3; i < 6; i++) {

```

Comment:
1. we can use only one loop
2. refactor this hardcoded condition to dynamically-based condition - calc middle figure from FIGURES_AMOUNT constant